src/index.js

```diff
@@ -11,17 +11,71 @@ const nodePathParts = [
 process.env.NODE_PATH = nodePathParts.join(nodePathDelim)
 
 const core = require('@actions/core')
+const fetch = require('node-fetch').default
 const childProcess = require('child_process')
 const lhciCliPath = require.resolve('@lhci/cli/src/cli.js')
 const input = require('./input.js')
 const output = require('./output.js')
 
 // audit urls with Lighthouse CI
 async function main() {
+  let status
   core.startGroup('Action config')
   console.log('Input args:', input)
   core.endGroup() // Action config
 
+  /*******************************WARMING UP***********************************/
+  if (input.netlifySite) {
+    core.startGroup('Warming up')
+    const retryNumber = 5
+    let runs = 0
+    const sleep = async () => {
+      return new Promise(r => setTimeout(r, 60000))
+    }
+    /**
+     * @return {Promise<number>}
+     */
+    const resolveNetlifyBuildURL = async () => {
+      runs = runs + 1
+      try {
+        const res = await Promise.race(
+          input.urls.map(async url => {
+            console.log(`Pinging Netlify site ${url}`)
+            return fetch(url)
+          })
+        )
+        if (res.status === 200) {
+          console.log('Netlify site finished build, continue audit...')
+          return Promise.resolve(0)
+        } else if (runs > retryNumber) {
+          console.log('No 200 response from Netlify')
```

Comment:
Good to be more specific.
```suggestion
          console.log('No HTTP 200 OK from Netlify within 5 minute timeout.')
```